Mods/EpipEncounters_7d32cb52-1cfd-4526-9b84-db4867bf9356/Story/RawFiles/Lua/Epip/QuickInventory/Filters/Equipment.lua

```diff
@@ -408,22 +420,75 @@ QuickInventory.Hooks.IsItemVisible:Subscribe(function (ev)
     ev.Visible = visible
 end)
 
--- Sort equipment by rarity.
-QuickInventory.Hooks.SortItems:Subscribe(function (ev)
-    if QuickInventory:GetSettingValue(QuickInventory.Settings.ItemCategory) == "Equipment" then
-        local rarityA, rarityB = ev.ItemA.Stats.Rarity, ev.ItemB.Stats.Rarity
-        local scoreA, scoreB
+---@param item EclItem
+---@return integer
+function ScoreLevel(item)
+    local result = item.Stats and item.Stats.Level
+    return result or 0
+end
 
-        if Item.IsArtifact(ev.ItemA) then
-            rarityA = "Artifact"
-        end
-        if Item.IsArtifact(ev.ItemB) then
-            rarityB = "Artifact"
+---@param item EclItem
+---@return integer
+function ScoreRarity(item)
+    local rarity = item.Stats and item.Stats.Rarity
+    if Item.IsArtifact(item) then
+        rarity = "Artifact"
+    end
+
```

Comment:
These days for short functions like these I don't leave a empty line before return